components/atom/tag/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@schibstedspain/sui-atom-tag",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rm -Rf ./lib && mkdir -p ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@s-ui/component-dependencies": "latest",
+    "@schibstedspain/sui-theme": "8"
```

Comment:
Now component dependencies provides sui-theme. No need to include it:
https://github.com/SUI-Components/sui/blob/master/packages/sui-component-dependencies/package.json#L13